lib/pages/home_screen/movie_list_card.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db_module2_part1/model/movie_data_model.dart';
+import '../../constants.dart';
+
+class MovieListCard extends StatelessWidget {
+  static const double movieCardBorderRadius = 5;
+  static const double movieCardPadding = 10;
+  static const double movieCardMargin = 5;
+  static const double movieTitlePaddingTop = 10;
+  static const double movieTitlePaddingBottom = 3;
+  static const double movieTitlePaddingLeft = 6;
+  static const double movieTitleFontSize = 22;
+  static const double movieOverviewFontSize = 13;
+  static const double movieOverviewPaddingLeft = 8;
+  static const double movieOverviewPaddingBottom = 15;
+  static const int movieOverviewMaxLines = 3;
+  static const double moviePosterHeight = 150;
+  static const double moviePosterWidth = 100;
+  static const String moviePosterPath = 'https://image.tmdb.org/t/p/original/';
+  static const double movieVoteAverageFontSize = 20;
+  static const double starIconSize = 35;
+  MovieDataModel movie;
+
+  MovieListCard(this.movie, {super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(movieCardBorderRadius,),
+        color: const Color(AppConstants.appAccentColor,),
+      ),
+      padding: const EdgeInsets.all(movieCardPadding,),
+      margin: const EdgeInsets.all(movieCardMargin,),
+      child: Row(
+        children: [
+          Image.network(
+            '$moviePosterPath${movie.posterPath}',
```

Comment:
You could do this interpolation as a getter method inside `MovieDataModel`:
`String get posterUrl => '$moviePosterPath${movie.posterPath}'`